node/tests/SharedTests.ts

```diff
@@ -6366,6 +6366,64 @@ export function runBaseTests<Context>(config: {
         },
         config.timeout,
     );
+
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
```

Comment:
I'm not convinced that XDEL needs it's own test... since it's easy to tack on to an existing test.  
Consider merging this test with another test moving forward.

Reply:
Will keep that in mind, thanks!